src/webnative.ts

```diff
@@ -1,9 +1,16 @@
 import * as webnative from 'webnative';
-import * as filecoin from '../webnative-filecoin/src/index';
-import type Wallet from '../webnative-filecoin/src/wallet';
+import * as filecoin from 'webnative-filecoin';
+import { Wallet } from 'webnative-filecoin';
 import { writable, Writable } from 'svelte/store';
 
 
+webnative.setup.debug({ enabled: true });
+webnative.setup.endpoints({
+  api: "https://runfission.net",
+  lobby: "http://localhost:8001",
+  user: "fissionuser.net"
```

Comment:
I know this is still under development ... but this won't work well on deploy. Might be worth toggling on `NODE_ENV` ?

Reply:
Oh yeah, good point! We should handle that and enabling debugging in the environment.